pycaret/time_series/forecasting/oop.py

```diff
@@ -3194,13 +3385,53 @@ def load_model(
 
         """
 
-        return super().load_model(
+        model = super().load_model(
             model_name=model_name,
             platform=platform,
             authentication=authentication,
             verbose=verbose,
         )
 
+        #### In time series, the fitted pipeline is needed to make predictions,
```

Comment:
Do this part inside `predict_model`

`predict_model` is an exception to other functions. It can take a model or a pipeline as input.
Move these steps to `predict_model` (don't set the pipeline since the loaded model may be using a different pipeline than the current experiment).

In `predict_model`
- (workflow without loading model) if it is a model and user has run setup, then add the `self.pipeline` to model and make the prediction
- (workflow with user loaded pipeline) if it is a Pipeline, just use it as is to make the prediction.
- (workflow with user loaded model) if it is a model --> resume old experiment - use `self.pipleine` to make prediction. Use Case: Resuming experiments - save `model_only` as if coming from `create_model`.
- (exception) if it is a model and user has not run setup, throw an exception